RELEASE/scripts/autoscend/combat/auto_combat_default_stage2.ash

```diff
@@ -125,6 +123,29 @@ string auto_combatDefaultStage2(int round, monster enemy, string text)
 		combatState += "(banishercheck)";
 	}
 
+
+	// Free run from monsters we want to banish but are unable to
+	if(auto_wantToBanish(enemy, my_location()))
+	{
+		//TODO: Make freeRunCombatString() like banish
+		//TODO use $item[Tattered Scrap Of Paper] as a free runaway?
+		//TODO use $item[Green Smoke Bomb] as a free runaway?
+
+		if (canUse($skill[Peel Out]))
+		{
+			int maxPeelOut = 10;
+			if (get_property("peteMotorbikeTires") == "Racing Slicks")
+			{
+				maxPeelOut = 30;
+			}
+			if (get_property("_petePeeledOut").to_int() < maxPeelOut)
```

Comment:
this could probably do with a function `int peelOutRemaining()` considering it is used twice at two different spots